aries-site/src/layouts/content/Example/Example.js

```diff
@@ -180,7 +180,10 @@ export const Example = ({
         width={width}
         ref={inlineRef}
       >
-        <ThemeContext.Extend value={scaledTheme || theme}>
+        {/* prevent theme from overriding the desired background color */}
+        <ThemeContext.Extend
+          value={{ ...(scaledTheme || theme), background: 'background' }}
```

Comment:
```suggestion
          value={{ ...(scaledTheme || theme), background: 'background-front' }}
```
Think we need to do `background-front` instead.

Reply:
For the way dark vs light mode differ